src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
```

Comment:
The image path (`../images/star.svg`) could be defined as an SCSS variable for better maintainability, especially if the path changes or is used in multiple places.